changelog/2021/MAY.rst

```diff
@@ -0,0 +1,11 @@
+
+--------------------------------------------------------------------------------
+                                      New                                       
+--------------------------------------------------------------------------------
+
+* IOSXE
```

Comment:
Please make sure your changelogs include what class was changed. You can follow this template: https://github.com/CiscoTestAutomation/genieparser/blob/master/changelog/undistributed/template.rst

Reply:
github is only showing the difference since the previous reviewer commented that it needed to be New not Fix. if you look at the complete file all the info is there.

Reply:
You have to add the name of the class you made to the changelog. For example you'd add ShowInterfaceTransceiverDetail under IOS. Take another look at some of the existing changelog files to get a sense of what I mean. 

Here's another example. https://github.com/CiscoTestAutomation/genieparser/blob/master/changelog/undistributed/changelog_kazemy_show_ip_nbar_version_20210513.rst

Reply:
These comments are appearing in the middle of the changed file, when i view the changes i have the new then IOSXE section.

Then below it i have the classes that i've added for IOSXE, and IOS section and the classes for IOS.

The original comment to this change was that i needed to use new not fix and it's broken up the view of the change log. if you view the change log in my repo it has exactly what you're asking for.

Reply:
Hi @ahusking have you pushed the commits with these changes? I dont see them. I noticed you updated `changelog/2021/MAY.rst` but for new changes we like to create a new file [here](https://github.com/CiscoTestAutomation/genieparser/tree/master/changelog/undistributed). Then come release time, we will combine all those files under the MAY.rst. This helps minimize merge conflicts.

Regarding the format, your changelog should look like this:
```
--------------------------------------------------------------------------------
                                      New                                       
--------------------------------------------------------------------------------

* IOSXE
    * Added ShowInterfaceTransceiver for:
        * show interfaces {interface} transceiver
* IOS
    * Added ShowInterfaceTransceiver for:
        * show interface {interface} transceiver
    * Added ShowInterfaceTransceiverDetail for:
        * show interface {interface} transceiver detail
        
--------------------------------------------------------------------------------
                                      Fix                                       
--------------------------------------------------------------------------------

* IOSXE
    * Modified ShowInterfaceTransceiverDetail to:
        * Parses transceiver information

Reply:
Changelog file looks good now